package/src/components/Channel/Channel.tsx

```diff
@@ -1653,6 +1653,8 @@ const ChannelWithContext = <
         // eslint-disable-next-line @typescript-eslint/no-unused-vars
         reaction_counts,
         // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        reaction_groups,
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
What is this `no-unused-vars` sorcery? :) Can we use `lodash/pick` instead?

Reply:
Legacy code. Let me see if this can be improved some way 😭

Reply:
Fixed using  lodash's `omit`